skyportal/handlers/api/source.py

```diff
@@ -1943,6 +1943,9 @@ def post_source(data, user_id, session, refresh_source=True):
                 auto_submission=True,
             )
             session.add(submission_request)
+            log(
+                f"Added TNSRobotSubmission request for obj_id {obj.id} saved to group {group.id} with tnsrobot_id {tnsrobot_group_with_autoreporter.tnsrobot_id} for user_id {user.id}"
```

Comment:
Not necessary to use here, but fun feature of f-strings: `f"{obj.id=}"` -> `obj.id=3`.

Reply:
oh!!! Where has this been all my life 💀 💀 💀 💀 💀 💀 

Thanks! That will be super super useful to shorten a lot of f strings!